docs/content/logging-and-monitoring/service-insight.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Service Insight
+
+description: "The Ingress Controller exposes the Service Insight endpoint."
+weight: 2100
+doctypes: [""]
+aliases:
+    - /service-insight/
+toc: true
+docs: ""
+---
+
+
+The Ingress Controller exposes an endpoint and provides host statistics for Virtual Servers.
+It exposes data in JSON format and returns HTTP status codes.
+The response body holds information about the total, down and the unhealthy number of
+upstreams associated with the hostname.
+Returned HTTP codes indicate the health of the upstreams (service).
+
+The service is not healthy (HTTP response code different than 200 OK) if all upstreams are unhealthy.
+The service is healthy if at least one upstream is healthy. In this case, the endpoint returns HTTP code 200 OK.
+
+
+
+## Enabling Service Insight Endpoint
+
+If you're using *Kubernetes manifests* (Deployment or DaemonSet) to install the Ingress Controller, to enable the Service Insight endpoint:
+1. Run the Ingress Controller with the `-enable-service-insight` [command-line argument](/nginx-ingress-controller/configuration/global-configuration/command-line-arguments). As a result, the Ingress Controller will expose the enpoint via the path `/probe/{hostname}` on port `9114` (customizable via the `-service-insight-listen-port` command-line argument).
```

Comment:
```suggestion
1. Run the Ingress Controller with the `-enable-service-insight` [command-line argument](/nginx-ingress-controller/configuration/global-configuration/command-line-arguments). This will expose the Ingress Controller endpoint via the path `/probe/{hostname}` on port `9114` (customizable with the `-service-insight-listen-port` command-line argument).
```